frontend/src/hooks/sails/nft/api.ts

```diff
@@ -0,0 +1,51 @@
+import { useAlert, useProgram, useSendProgramTransaction } from '@gear-js/react-hooks';
+
+import { useMarketplace } from '@/context';
+import { SailsProgram } from '@/hooks/sails/nft';
+
+export function useProgramInstance(programId?: `0x${string}`) {
```

Comment:
```suggestion
export function useProgramInstance(programId: HexString | undefined) {
```

There's `HexString` in `@gear-js/api`. Also, better to be clear that it's required parameter with nullable value - otherwise it's easy to forget to pass it.

Reply:
done